### PR TITLE
add layout and styles

### PR DESCRIPTION
**Pull request by nap4sov**

### CODE REVIEW

File: src/helpers/formValidation.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import { usernameRegex, emailRegex } from '../constants/regex';
+import { errorMessages } from '../constants/statusMessages';
+
+// username must consist of letters and spaces only
+// email must be a valid email
+// password must be 6 symbols or more
+
+export const validateLogin = ({ email, password }) => {
+    console.log(email, password);
```
Comment (Beinyk) [inline src/helpers/formValidation.js]: remove console.log

File: src/components/Forms/LoginForm/index.jsx [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+        setFormData({ ...formData, [target.name]: target.value });
+
+    const handleSubmit = event => {
+        event.preventDefault();
+
+        const validationErrors = validateLogin(formData);
+        setErrors(validationErrors);
+
+        if (loginFailed(validationErrors)) return;
+
```
Comment (Beinyk) [inline src/components/Forms/LoginForm/index.jsx]: combine in one function validateLogin (which return {error, data})

File: src/helpers/formValidation.js (hunk 1 above)
Comment (nap4sov) [reply src/helpers/formValidation.js]: 👍

File: src/components/Forms/LoginForm/index.jsx (hunk 2 above)
Comment (nap4sov) [reply src/components/Forms/LoginForm/index.jsx]: 👍

**Pull request merged**